client/lib/analytics/index.js

```diff
@@ -574,6 +586,11 @@ const analytics = {
 			label,
 			value
 		) {
+			if ( 'undefined' !== typeof value && /^[0-9.]+$/.test( String( value ) ) ) {
+				value = Math.round( Number( String( value ) ) ); // GA requires an integer value.
+				// https://developers.google.com/analytics/devguides/collection/analyticsjs/field-reference#eventValue
+			}
+
```

Comment:
Doh, this is really silly, what use do we have of values that can't have fractional values? I guess this is one extra vote to then implement the Enhanced Ecommerce API to pass proper values, I wonder if they did it on purpose. 

We could multiply everything by 100x but sounds like that'd be one extra source of confusion.

Reply:
Yeah, it seems kind of strange that it doesn't support floats, and even stranger that it doesn't just do the conversion itself as opposed to rejecting the event entirely.

It remains to be seen if this actually does solve the problem. At this point it's just an educated guess based on a few discussions (e.g., [this one](https://www.en.advertisercommunity.com/t5/Google-Analytics-Tracking-Goals/Event-Tracking-Value-Why-Integer-Values-Only/m-p/482841#M499)) I read through and the fact that it seems to be the only issue I can find in this particular event. What remains a mystery is why we have even just the small number of impressions on this event at all, though. I guess there are a few slipping through and GA is counting them in some way even though rejecting all the others.